apps/playground/src/components/Comps/TPopover.vue

```diff
@@ -60,6 +60,15 @@
           <template #default> <b>Tooltip</b> content </template>
         </BTooltip>
       </BCol>
+      <BCol>
+        <BPopover click :placement="popoverPlacement">
+          <template #title> Click </template>
+          <template #target>
+            <BButton>click longlist</BButton>
+          </template>
+          <div v-for="(_, i) in Array(200)" :key="i">foobar {{ i }}</div>
```

Comment:
```suggestion
          <div v-for="i in 200" :key="i">foobar {{ i }}</div>
```
Should be valid syntax. Not that I'm super picky on the test environments, just trying to teach you about it if you didn't know about it

Reply:
oh, didn't know that, thanks! 😄